lib/modules/manager/composer/artifacts.ts

```diff
@@ -29,17 +29,43 @@ import {
   getPhpConstraint,
   requireComposerDependencyInstallation,
 } from './utils';
+import { GitTagsDatasource } from '../../datasource/git-tags';
+import type { HostRuleSearch } from '../../../util/host-rules';
+
+function findGithubPersonnalAccessToken(
+  search: HostRuleSearch
+): string | undefined {
+  const token = hostRules.find(search)?.token?.replace('x-access-token:', '');
```

Comment:
```suggestion
  const token = hostRules.find(search)?.token;
```
Pat should never have that prefix